### PR TITLE
Added function 'is-looping' from Processing API to quil. Documentatio…

### PR DESCRIPTION
**Pull request by NightMachinary**
…n NOT added (aside from the docstring). Should it be 'is-looping?'?
Solves https://github.com/quil/quil/issues/243.

### CODE REVIEW

File: src/cljc/quil/core.cljc [hunk 1]
```diff
@@ -1867,6 +1867,19 @@
...
+  ^{:requires-bindings true
+    :processing-name "isLooping()"
+    :category "Environment"
+    :subcategory nil
+    :added "2.7.2"}
+  is-looping
+  "Returns whether the sketch is looping."
+  []
+  #?(:clj (.isLooping (ap/current-applet))
+     :cljs (.-__isLooping (ap/current-applet))))
```
Comment (nbeloglazov) [inline src/cljc/quil/core.cljc]: Have you tested it with processing.js?

File: src/cljc/quil/core.cljc [hunk 2]
```diff
@@ -1867,6 +1867,19 @@
   #?(:clj (.frameRate (ap/current-applet))
      :cljs (.-__frameRate (ap/current-applet))))
 
+(defn
+  ^{:requires-bindings true
+    :processing-name "isLooping()"
+    :category "Environment"
+    :subcategory nil
+    :added "2.7.2"}
+  is-looping
```
Comment (nbeloglazov) [inline src/cljc/quil/core.cljc]: Canonical clojure way would would be `looping?`

**Comment by NightMachinary** [comment]
@nbeloglazov I used `@(:looping? (meta (qa/current-applet)))` for CLJS and fixed the name.

**Comment by nbeloglazov** [comment]
Thanks for the pull request!

**Pull request merged**